resources/assets/js/components/H2Components/Dialog.tsx

```diff
@@ -1,14 +1,16 @@
 import * as React from "react";
-import {
-  h2DialogResizeOnViewport,
-  h2DialogTrigger,
-  h2DialogEnableOpenDialogs,
-} from "@hydrogen-design-system/system/dist/import/latest/components/dialog/scripts/dialog";
-import { GeneralProps, GeneralBtnProps } from "./utils";
-
-interface DialogContext extends GeneralProps {
+import { focusOnElement, getTabList } from "../../helpers/forms";
+import { GeneralBtnProps } from "./utils";
+
+type Div = React.HTMLAttributes<HTMLDivElement>;
+type Heading = React.HTMLAttributes<HTMLDivElement>;
```

Comment:
Two identical constants seems unnecessary :P I also find the names just a bit confusing - could we call it DivProps instead?

Reply:
That is a mistake haha :p